benchmarks/benchmarks/stats.py

```diff
@@ -41,6 +41,18 @@ def time_barnard_exact(self, alternative):
         resBarnard = stats.barnard_exact(self.a, alternative=alternative)
 
 
+class ANOVAFunction(Benchmark):
+    def setup(self):
+        np.random.seed(12345678)
```

Comment:
Thanks! I found in this file all the functions used the old style, will we modify them all to the new-style? If so, I can help do the modification :)

Reply:
I think modifying all `numpy.random` usage to the new API in a follow up PR would be nice. If we leave it as a mix or mostly old-style, others are going to copy that old style.

There's a very minor issue that for some benchmarks, changing the exact set of random numbers that is generated may affect the timing. This can be detected by using `--bench-compare`, to see that the results didn't change significantly after making the change. In case there's one or two benchmarks where changed timings are significant, those could then be left as old-style.